janitor/functions.py

```diff
@@ -289,8 +320,8 @@ def convert_excel_date(df, column):
     :param str column: A column name.
     :returns: A pandas DataFrame with corrected dates.
     """
-    df[column] = (pd.TimedeltaIndex(df[column], unit='d')
-                  + dt.datetime(1899, 12, 30))
+    df[column] = (pd.TimedeltaIndex(df[column], unit='d') +
+                  dt.datetime(1899, 12, 30))
```

Comment:
Logical and mathematical operators are put on a new line for clarity (particularly if there are multiple math operations going on). Please revert this.

Reply:
Sure. Sorry, I only changed it as I was getting these two cautions from flake8.
```
janitor/functions.py:223:15: W503 line break before binary operator
janitor/functions.py:323:19: W503 line break before binary operator
```
Reverted to operator on newline.